schema/github.schema.json

```
@@ -168,6 +168,11 @@
           "description": "Experimental: If set, configures hours cached permissions from teams and organizations should be kept for. Setting a negative value disables syncing from teams and organizations, and falls back to the default behaviour of syncing all permisisons directly from user-repository affiliations instead. [Learn more](https://docs.sourcegraph.com/admin/repo/permissions#teams-and-organizations-permissions-caching).",
           "type": "number",
           "default": 72
+        },
+        "internalRepos": {
```

```suggestion
        "internal": {
```

as per your proposal https://github.com/sourcegraph/sourcegraph/issues/34684 , and I think it makes sense with my other comment

---

I initially went with internal but then renamed this to internalRepos as the former felt less intuitive when reading its usage. How do you feel about it?

---

I think internal makes sense: "everything syncd from this code host has internal visibility"

---

I'm a bit confused here, my understanding of "internal repos" and "making all synced repos are internally visible" are two different things. Which are we addressing? If I understand correctly from https://github.com/sourcegraph/sourcegraph/issues/34684, we are addressing the former? But this PR seems addressing the latter?